builtins.go

```diff
@@ -23,6 +23,10 @@ var builtins = []Func{
 	{Name: "gopkg.in/yaml.v3.Unmarshal", Tag: "yaml", ArgPos: 1},
 	{Name: "(*gopkg.in/yaml.v3.Encoder).Encode", Tag: "yaml", ArgPos: 0},
 	{Name: "(*gopkg.in/yaml.v3.Decoder).Decode", Tag: "yaml", ArgPos: 0},
+	{Name: "gopkg.in/yaml.v2.Marshal", Tag: "yaml", ArgPos: 0},
```

Comment:
Haven't seen `v2` being imported in a while, is it still in use? We should probably add these in a separate PR covered with tests :)

Reply:
yes, I just added that because during my test the imports were wrong.